queries/prql/highlights.scm

```diff
@@ -0,0 +1,147 @@
+[
+  (keyword_from)
+  (keyword_filter)
+  (keyword_derive)
+  (keyword_group)
+  (keyword_aggregate)
+  (keyword_sort)
+  (keyword_take)
+  (keyword_window)
+  (keyword_join)
+  (keyword_select)
+  (keyword_switch)
+  (keyword_append)
+  (keyword_remove)
+  (keyword_intersect)
+  (keyword_rolling)
+  (keyword_rows)
+  (keyword_expanding)
+  (keyword_let)
+  (keyword_prql)
+  (keyword_from_text)
+] @keyword
+
+[
+ (literal)
+ (f_string)
+ (s_string)
+] @string
+
+(assignment
+  alias: (field) @field)
+
+alias: (identifier) @field
+
+
+
+(comment) @comment @spell
+
+(keyword_func) @keyword.function
+
+(function_call
+  (identifier) @function.call)
+
+[
+  "+"
+  "-"
+  "*"
+  "/"
+  "="
+  "=="
+  "<"
+  "<="
+  "!="
+  ">="
+  ">"
+  "->"
```

Comment:
`->` could be `@punctuation.delimiter`, just commenting to double check

Reply:
I think `@operator` fits quite well.

Reply:
I read the PRQL book a bit, `->`doesn't operate on anything, but more delimits function parameters and the body, and switch statement cases. So I think it fits better in `@punctuation.delimiter`

Reply:
Yes, you are probably right. I changed it.

Reply:
Yeah, I think C/C++ are the only languages I can think of off the top of my head where -> is actually an operator, oftentimes it's a delimiter :)